src/enclave-agent/src/main.rs

```diff
@@ -45,7 +48,16 @@ async fn main() -> Result<()> {
         TCP_SOCK_ADDR
     };
 
-    let image_service = Box::new(ImageService::new()) as Box<dyn image_ttrpc::Image + Send + Sync>;
+    let dc = if let Some(file_path) = matches.value_of("decrypt-config") {
+        DecryptConfig::load_from_file(&file_path.to_string())?
```

Comment:
this path should handle errors better

Reply:
Sure. The error will be handled in the `load_from_file`, this path will raise the error if failed.